tests/system/Database/Live/SQLite3/GetFieldDataTest.php

```diff
@@ -32,10 +32,19 @@ protected function createForge(): void
             'database' => 'database.db',
             'DBDebug'  => true,
         ];
-        $this->db    = db_connect($config);
+        $this->db    = db_connect($config, false);
         $this->forge = Database::forge($config);
     }
 
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+
+        if (is_file(WRITEPATH . 'database.db')) {
+            unlink(WRITEPATH . 'database.db');
+        }
```

Comment:
You should not remove the database file in use.
It causes "SQLite3::exec(): attempt to write a readonly database"
because the file is gone, but the connection remains.

Try:
```
$ vendor/bin/phpunit tests/system/Database/
```